drf_braces/fields/custom.py

```diff
@@ -84,5 +85,47 @@ def validate_precision(self, data):
         return data
 
 
+class RoundedDecimalField(fields.DecimalField):
+    """
+    Currency field subclass of Decimal used for rounding currencies
+    to two decimal places.
+    """
+    rounding = None
+
+    def __init__(self, max_digits=None, decimal_places=2, rounding=None, *args, **kwargs):
+        max_digits = max_digits or self.MAX_STRING_LENGTH
```

Comment:
can be safely removed as `quantize` does not add the `context.prec` when max digits is `None`. it was to prevent exceptions with older versions of DRF